perun-base/src/main/resources/perun-roles.yml

```diff
@@ -2150,6 +2150,16 @@ perun_policies:
     include_policies:
       - default_policy
 
+  getAllAllowedGroupsToHierarchicalVo_Vo_policy:
+    policy_roles: []
```

Comment:
Since this is a READ method, we can include here PERUNOBSERVER as well, same for the other method.